examples/cli-tricky-directories/A directory name with spaces /CNAME

```diff
@@ -0,0 +1 @@
+a-directory-name-with-spaces
```

Comment:
I was really confused why we were making DNS entries for these until I read the rest of the code :)

Reply:
CNAME was a BADNAME.